x/mongo/driver/operation_test.go

```diff
@@ -149,7 +150,8 @@ func TestOperation(t *testing.T) {
 			want Type
 		}{
 			{"deployment doesn't support", Operation{Deployment: deploymentNoRetry}, description.Server{}, Type(0)},
```

Comment:
This seems like an outdated test case given that deployments are no longer used to determine retryability. `deploymentNoRetry` is of type `mockDeployment` with the `returns.retry` flag set to false, so maybe you can just get rid of the `returns.retry` flag altogether (and you'll probably remove the `SupportsRetryWrites` method when addressing one of my earlier comments).

Reply:
done